### PR TITLE
[NYC-2019] Add venue address

### PR DESCRIPTION
**Pull request by TomOnTime**
Add venue location.

### CODE REVIEW

File: data/events/2019-new-york-city.yml [hunk 1]
```diff
@@ -28,14 +28,14 @@ registration_link: "" # If you have a custom registration link, enter it here. T
 
 # Location
 #
-coordinates: "" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+coordinates: "40.757933 -73.986429" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
 location: "new-york-city" # Defaults to city, but you can make it the venue name.
```
Comment (bridgetkromhout) [inline data/events/2019-new-york-city.yml]: You probably want to change this to the venue name.

**Comment by TomOnTime** [comment]
Ok. This is ready to merge. Thanks!

**Comment by bridgetkromhout** [comment]
Do you want to list the address twice? See https://deploy-preview-5821--devopsdays-web.netlify.com/events/2019-new-york-city/welcome/ for what I mean.

**Comment by bridgetkromhout** [comment]
This is now taken care of in https://github.com/devopsdays/devopsdays-web/pull/5824.

**Pull request closed without merging**